packages/react-dom/src/__tests__/ReactDOMServerIntegrationHooks-test.internal.js

```diff
@@ -613,8 +613,114 @@ describe('ReactDOMServerHooks', () => {
   });
 
   describe('useContext', () => {
+    itThrowsWhenRendering(
```

Comment:
The diff confused git. This is not a new test.

All I did is move the `useContext()` test with bits into `readContext()` section below.